content/en/sponsor-kcd-helsinki-2025/_index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Sponsor KCD Helsinki 2025!
+linkTitle: Sponsor KCD Helsinki 2025!
+toc_hide: true
+list_pages: false
+menu:
+  main:
+    weight: 5
+description: Sponsor KCD Helsinki 2025!
+---
+
+## What is this?
+
+We’re organizing the first-ever Kubernetes Community Days (KCD) conference in Helsinki, and it’s happening on
+**May 6, 2025**, in **Helsinki**! This event is part of the global KCD program, supported by the Cloud Native Computing
+Foundation (CNCF), and is a big step forward in building a stronger cloud-native community here in Finland. Our
+inclusion in the program has been officially announced here.
+
```

Comment:
No link in announcement

Reply:
Fixed in [0240c38](https://github.com/cloud-native-finland/cloud-native-finland-site/pull/9/commits/0240c3808b584eefe965017632703d330b7e0c56)